draft-ietf-trans-rfc6962-bis.md

```diff
@@ -1411,14 +1384,16 @@ Outputs:
 : entries:
   : An array of objects, each consisting of
 
-    leaf_input:
+    log_entry:
     : The base64 encoded `TransItem` structure of type `x509_entry_v2` or
       `precert_entry_v2` (see {{tree_leaves}}).
```

Comment:
Why reference {{tree_leaves}} here?  Wouldn't it be more appropriate to reference {{log_entries}} ?